rhel/vex/parser.go

```diff
@@ -0,0 +1,410 @@
+package vex
+
+import (
+	"bufio"
+	"bytes"
+	"context"
+	"errors"
+	"fmt"
+	"io"
+	"strings"
+
+	"github.com/klauspost/compress/snappy"
+	"github.com/package-url/packageurl-go"
+	"github.com/quay/zlog"
+
+	"github.com/quay/claircore"
+	"github.com/quay/claircore/toolkit/types/cpe"
+	"github.com/quay/claircore/toolkit/types/csaf"
+	"github.com/quay/claircore/toolkit/types/cvss"
+)
+
+// Parse implements [driver.Updater].
+func (u *VEXUpdater) Parse(ctx context.Context, contents io.ReadCloser) ([]*claircore.Vulnerability, error) {
```

Comment:
For StackRox's usage of Claircore at the moment, we pretty much start fresh every time. Does not implementing `Parse` mean our use case may not work? Related: will the jsonblob stuff work OOTB?

Reply:
When starting from fresh, `DeltaParse` will consider the whole RH VEX corpus, so it should be fine there. I think for the jsonblob store to work the `DeltaUpdateVulnerabilities` would need to be implemented.

Reply:
https://github.com/quay/claircore/pull/1345 I added this do basically make sure any delta updaters will persist all their data.